### PR TITLE
Nstonehouse/us71024/transplant styles daylight cr

### PR DESCRIPTION
**Pull request by njostonehouse**

### CODE REVIEW

File: htmlblock.html [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+				font-family: Monospace;
+				font-size: 13px;
+				margin: 13px 0;
+			}
+
+			.vui-typography .d2l-htmlblock p {
+				margin: 0.5em 0 1em 0;
+			}
+
+			[dir='rtl'] .vui-typography .d2l-htmlblock {
```
Comment (dbatiste) [inline htmlblock.html]: Is this working?  Note that `[dir='rtl']` and `vui-typography` are almost always on the same element.

File: htmlblock.html [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+				line-height: normal;
+				text-align: left;
+				overflow-x: auto;
+				overflow-y: hidden;
+			}
+
+			.vui-typography .d2l-htmlblock ul,
+			.vui-typography .d2l-htmlblock ol {
+				list-style-position: outside;
+				margin: 1em 0;
```
Comment (dbatiste) [inline htmlblock.html]: Are more changes coming for this? Thinking about the use if `em` values here.

File: htmlblock.html [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+			.vui-typography .d2l-htmlblock ul[type="disc"] {
+				list-style-type: disc;
+			}
+
+			.vui-typography .d2l-htmlblock ul ul,
+			.vui-typography .d2l-htmlblock ul ol,
+			.vui-typography .d2l-htmlblock ol ul,
+			.vui-typography .d2l-htmlblock ol ol {
+				margin-top: 0;
+				margin-bottom:0;
```
Comment (dbatiste) [inline htmlblock.html]: Nitpick: should be space after `:`.

**Comment by dbatiste** [comment]
My understanding is that this is simply a transplant - more changes coming eventually to fix the styles to follow typography, yes?

**Comment by njostonehouse** [comment]
Yes, it's just a transplant, although I don't have any plans currently to adjust the styles, I agree it needs to be done.

File: htmlblock.html (hunk 1 above)
Comment (njostonehouse) [inline htmlblock.html]: That's an excellent point. I hadn't checked. Will test.

File: htmlblock.html (hunk 2 above)
Comment (njostonehouse) [inline htmlblock.html]: More changes should come, yes. Nothing in progress, but definitely a good point.

Comment (dbatiste) [inline htmlblock.html]: Might be worthwhile creating a story to track (if one doesn't already exist).

Comment (njostonehouse) [inline htmlblock.html]: Yup, I put @joshassad on it.

**Pull request closed without merging**